aspnetcore/includes/managed-identities-test-non-production.md

```diff
@@ -0,0 +1,4 @@
+> [!NOTE]
+> Microsoft recommends that you use the most secure authentication flow available. The authentication flow described in this procedure is only suitable for testing and local development.
+>
+> Azure SQL Database should use [Managed Identities for Azure resources](/sql/connect/ado-net/sql/azure-active-directory-authentication#using-managed-identity-authentication) instead of the flow described in this tutorial. For non-Azure apps, use a secure authentication flow similar to [Managed Identities for Azure resources](/sql/connect/ado-net/sql/azure-active-directory-authentication#using-managed-identity-authentication).
```

Comment:
```suggestion
> Azure SQL Database should use [Managed Identities for Azure resources](/sql/connect/ado-net/sql/azure-active-directory-authentication#using-managed-identity-authentication) instead of the flow described in this tutorial. For non-Azure apps, use a secure authentication flow similar to managed identities for Azure resources.
```